### PR TITLE
ENV and spring/bundler autodetermination

### PR DESCRIPTION
**Pull request by astrauka**
Ability to set environment variables for the run.
Autodetermine whether to use bundler or spring. Do not use both at the same time.

### CODE REVIEW

**Comment by astrauka** [comment]
@maltize tagging you in case you did not get notified about the pull request

**Comment by astrauka** [comment]
@gs @gja tagging you in case you did not get notified about the pull request

**Comment by rposborne** [comment]
+1 currently the check for spring flag will just crash if spring is not present.

File: Default (Linux).sublime-keymap [hunk 1]
```diff
@@ -1,21 +1,21 @@
 [
-  { "keys": ["ctrl+shift+r"], "command": "run_single_ruby_test",
+  { "keys": ["super+shift+r"], "command": "run_single_ruby_test",
```
Comment (astrauka) [inline Default (Linux).sublime-keymap]: @rposborne will also revert this file

**Comment by astrauka** [comment]
I think that my code is far from perfect and it fails when first project path is not the project root path, so closing for now

**Pull request closed without merging**